go/vt/vtgate/vschemaacl/vschemaacl.go

```diff
@@ -34,19 +37,35 @@ var (
 	acl map[string]struct{}
 )
 
+// RegisterSchemaACLFlags installs log flags on the given FlagSet.
+//
+// `go/cmd/*` entrypoints should either use servenv.ParseFlags(WithArgs)? which
+// calls this function, or call this function directly before parsing
+// command-line arguments.
+func RegisterSchemaACLFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&AuthorizedDDLUsers, "vschema_ddl_authorized_users", AuthorizedDDLUsers, "List of users authorized to execute vschema ddl operations, or '%' to allow all users.")
+}
+
+func init() {
+	// register flag
+	for _, cmd := range []string{"vtexplain", "vtcombo", "vtgate", "vtgateclienttest"} {
```

Comment:
on second thought ... are we sure that `vtexplain` actually needs ACLs? i had the understanding it was entirely a local simulation, so you shouldn't need to be "authorized"

Reply:
I am not sure ... I added it because vtexplain help file has it so I added it to the array of cmd